src/main/java/core/basesyntax/strategy/MyFileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyFileReaderImpl implements MyFileReader {
+    public List<String> read(String filePath) {
```

Comment:
Method implementations should not start with an empty line. Please remove the empty line to adhere to the coding standards.